app/enterprise/changelog.md

```diff
@@ -4,6 +4,167 @@ no_search: true
 no_version: true
 skip_read_time: true
 ---
+## 2.3.0.0
+**Release Date** 2020/02/10
+
+### **Kong Gateway (Enterprise)**
+The following sections list {{site.ee_product_name}}-exclusive updates,
+features, and fixes for the **2.3.0.0** version.
+
+Kong Enterprise is renamed to Kong Gateway, or Kong Gateway (Enterprise) going forward.
+
+### Features
+
+#### Enterprise
+- Kong Gateway can now run in free mode, without a license, which gives users access to Kong Manager and an easy upgrade path to Enterprise.
+- A (non-OSS) Kong Gateway running in its default Free Mode can be upgraded to add
+Enterprise functionality modules and features including **Kong Vitals**, **RBAC**,
+**Workspaces**, and **Enterprise Plugins** simply by applying a new Kong license.
+Contact your Kong sales representative for more information.
```

Comment:
A net new user that starts off in free mode is not going to have a sales rep that they know about or even know how to contact us.  Let's link them to https://konghq.com/get-started/ and instead of telling them to contact their sales rep, let's say "Reach out to us if you'd like to learn more" as the text of the link